scripts/status_consistency.py

```diff
@@ -0,0 +1,36 @@
+from app import geo
+from app.firebase import db
+
+from app.constants import locationsTable, venuesTable, GPS_LOCATIONS
+
+# The casing of tripadvisor is not consistent across our Firebase scripts
+TA_PROVIDER = "tripadvisor"
+TA_DETAILS = "tripAdvisor"
+
+dryRun = True
+
+def updateStatus(center, radius_km, provider, detailsProvider, version):
```

Comment:
When might you want to call this fn? It seems to set the provider status to the given value, but why would you ever want to do that? Isn't the status set when we crawl? Maybe add a fn, or file, comment clarifying why you wrote this and why someone might want to run this.